plg_iwcsocialmeta/forms/iwcsocialmeta.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
As I have explained, please move files to folder structure https://github.com/InstaWebCare/SocialMetaTag/tree/master/plugins/content/iwcsocialmeta 
For Example: here folder `forms` and other files should be start from [here](https://github.com/InstaWebCare/SocialMetaTag/tree/master/plugins/content/iwcsocialmeta)